packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -326,9 +328,17 @@ Future<void> _buildGradleProjectV2(
     FlutterProject flutterProject,
     String gradle,
     BuildInfo buildInfo,
-    String target) async {
+    String target,bool isBuildingBundle) async {
   final GradleProject project = await _gradleProject();
-  final String assembleTask = project.assembleTaskFor(buildInfo);
+
+  String assembleTask;
+
+  if(isBuildingBundle){
+    assembleTask = project.bundleTaskFor(buildInfo);
+  }else{
```

Comment:
nit: please use the same format as other places in the file (e.g. argument goes on the next line, spaces after `if` and before `{` and around `else`, etc)